### PR TITLE
Add community coordination implementations(consul and redis) on README

### LINKED ISSUE #108: Mention contructs-coordination-consul in the README
https://github.com/Tecsisa/constructr-consul

Of high quality and uses Akka HTTP to talk to Consul.

### PR DESCRIPTION
**Pull request by everpeace**
this fixes #108

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,12 @@ constructr {
 
 ConstructR comes with out-of-the-box support for etcd: simply depend on the "constructr-coordination-etcd" module. If you want to use some other coordination backend, e.g. Consul, simply implement the `Coordination` trait from the "constructr-coordination" module and make sure to provide the fully qualified class name via the `constructr.coordination.class-name` configuration setting.
 
+### Community Coordination Implementations
```
Comment (hseeberger) [inline README.md]: Blank line after the heading, please.

File: README.md [hunk 2]
```diff
@@ -128,6 +128,12 @@ constructr {
...
+There are some implementations for other coordination backend than etcd.
```
Comment (hseeberger) [inline README.md]: s/backend/backends and use colon instead of dot.

File: README.md [hunk 3]
```diff
@@ -128,6 +128,12 @@ constructr {
...
+
+* [Tecsisa/constructr-consul](https://github.com/Tecsisa/constructr-consul): This library enables to use Consul as cluster coordinator in a ConstructR based node.
```
Comment (hseeberger) [inline README.md]: I don't think "node" is a good term here. I suggest to either remove "in a ... node." or s/node/cluster.

File: README.md [hunk 4]
```diff
@@ -128,6 +128,12 @@ constructr {
...
+* [everpeace/constructr-redis](https://github.com/everpeace/constructr-redis): This library enables to use Redis as cluster coordinator in a ConstructR based node.
```
Comment (hseeberger) [inline README.md]: Like above.

**Comment by everpeace** [comment]
@hseeberger Thank you for elaboration!  I corrected points which you pointed out and pushed.

**Comment by hseeberger** [comment]
Great! Thanks a lot for your contribution(s)!

**Pull request merged**